docs/rules/avoid-dispatching-multiple-actions-sequentially.md

```diff
@@ -1 +1,60 @@
 # avoid-dispatching-multiple-actions-sequentially
+
+> This rule promotes disallows dispatching multiple actions one after another.
+
+## Rule Details
+
+An action should be an event which abstracts away the details of store internals.
+An action can be a composition of several events, in which case the developer might be tempted to dispatch several actions in sequence. But a better approach is t dispatch one "combining" action, which exactly describes what that event entails, and then map it to several other actions in an Effect
+
+Examples of **incorrect** code for this rule:
+
+```ts
+export class Component implement OnInit {
+  constructor(
+    private readonly store: Store<AppState>,
+  ) {}
+
+  ngOnInit() {
+    this.store.dispatch(loadEmployeeList());
+    this.store.dispatch(loadCompanyList());
+    this.store.dispatch(cleanData());
```

Comment:
Should we highlight that this is the problem here?

```suggestion
   // ⚠ multiple actions dispatched
    this.store.dispatch(loadEmployeeList());
    this.store.dispatch(loadCompanyList());
    this.store.dispatch(cleanData());
```

Reply:
Yes, I agree